src/main/java/org/isf/video/gui/PatientPhotoPanel.java

```diff
@@ -110,10 +114,21 @@ public PatientPhotoPanel(final PatientInsertExtended patientFrame, final Integer
 
 			externalPanel.add(box, BorderLayout.NORTH);
 			photoboothPanelPresentationModel.addBeanPropertyChangeListener(PhotoboothPanelModel.PROPERTY_IMAGE, propertyChangeEvent -> {
-				final BufferedImage newImage = (BufferedImage) propertyChangeEvent.getNewValue();
-				if (newImage != null) {
-					externalPanel.updatePhoto(ImageUtil.scaleImage(newImage, 160, 160));
-					patientFrame.setPatientPhoto(newImage);
+				
+				//ByteArrayOutputStream baos = new ByteArrayOutputStream();
+				//BufferedImage bia = (BufferedImage) propertyChangeEvent.getNewValue();
+				try {
+					//ImageIO.write(bia, "jpg", baos);
+					//BufferedImage bufferedImage = ImageIO.read(new ByteArrayInputStream(baos.toByteArray()));
+					//BufferedImage buffImg = new BufferedImage(bufferedImage.getWidth(), bufferedImage.getHeight(), bufferedImage.getType());
+					BufferedImage bi = (BufferedImage) propertyChangeEvent.getNewValue();
+				if (bi != null) {
+					externalPanel.updatePhoto(ImageUtil.scaleImage(bi, 160, 160));
+						patientFrame.setPatientPhoto(ImageUtil.fixImageFileSize(bi, MAXPROFPICFILESIZEBYTES, "png"));
+				} }catch (IOException e1) {
+					LOGGER.error("Oooops! Can't resize profile picture.");
+					// TODO Auto-generated catch block
+					e1.printStackTrace();
```

Comment:
Why the single `printStackTrace()`?

Reply:
maybe for debugging purposes since it's a new feature... anyway removed all comments and reformatted the code